_data/changelogs/utilities.yml

```diff
@@ -2,6 +2,19 @@ title: Utilities
 type: utility
 changelogURL:
 items:
+  - date: NNN-NN-NN
+    summary: Updated sass map instructions to reflect new pattern.
+    summaryAdditional: Users are now instructed to define only the variable they wish to change instead of having to define the entire map
```

Comment:
```suggestion
    summaryAdditional: Users are now instructed to define only the variable they wish to change instead of having to define the entire map.
```